### ISSUE TITLE
HTC Vive Tracker is not giving 3.3V after upgrading the firmware of the device. (Vive Tracker + Unity)

### ISSUE THREAD

**Issue by Cresset50c1** [ValveSoftware/steamvr_unity_plugin#224]
After upgrading the firmware the tracker is giving 0.03V (300 mili volts) instead of 3.3V. All of the sudden my gun vibration stopped working.

**Comment by zite** [comment]
Looks like htc manages this feature of firmware. Unfortunately you’ll have to contact them for a fix.

**Comment by CarlosMeloStuff** [comment]
I have had the same problem for months, I have 4 trackers and I have tested the haptic feedback on Arizona sunshine and other games that support the hyperblaster and used to have haptic feedback for the trackers and also in Unity an I can't get it to work. I have open the hyperblaster and confirm that the vibrator motor works correctly by connecting it to an external power source so the problem seems to be the steamvr software.

I have tried contacting Vive and Steam developers but they haven't responded.
